program/Cargo.toml

```diff
@@ -6,15 +6,16 @@ license = "WTFPL"
 publish = false
 
 [dependencies]
-solana-program = "1.6.7"
+solana-program = "1.6.12"
+solana-validator = { version = "1.6.12", optional = true }
 
 [features]
-test-bpf = []
+test-bpf = ["solana-validator"]
```

Comment:
solana-validator blows up compilation time, #22 uses ProgramTest even for cargo test-bpf